src/components/network/network-panel.js

```diff
@@ -137,6 +147,14 @@ genotet.NetworkPanel.prototype.setEdgeInfo_ = function(edge, container) {
     .text(edge.target.label);
   container.children('#weight').children('span')
     .text(edge.weight);
+  container.children('#weight').children('#rm-edge').click(function() {
+    this.signal('update', {
+      type: 'delete-edge',
+      source: edge.source.label,
+      target: edge.target.label
+    });
+    this.container.find('#edge-info').hide().slideUp();
```

Comment:
It seems to me that hide() will hide the element immediately so that slideUp has no effect. You can just use slideUp() I think.